### DISCUSSION TITLE
Wrong Device after interview

### DISCUSSION THREAD

**Discussion by GJayBoi** [Koenkk/zigbee2mqtt#19325] [Devices]
Hi,
Wondered if anyone else has seen this and if you can manually force what a device is, I have LOADS of these:
[AJ_ZB_GU10](https://www.zigbee2mqtt.io/devices/AJ_ZB_GU10.html)

But they show up once configured as these:
[TS0505B](https://www.zigbee2mqtt.io/devices/TS0505B_1.html#tuya-ts0505b_1)

They seem to be pretty much the same, so I am sure i'm not missing out on any functionality, but then OTA updates don't show at all for these and I wondered if that was why maybe?

Lastly, I have one of said bulbs that has not completed it's interview, but it all works, everything is configured... but is there a way I can force this interview again, or just change the status to true, the warning triangle next to it catches my eye every time I go into 2mqtt and it just bugs me... it's a pain to reset the device as the bulb is in the eaves of the house... and I can only turn off the whole circuit... and I don't really want to factory reset them all!

TIA,
JC.

**Comment by francisp2** [comment]
Did you try 'reconfigure' ?

**Reply by GJayBoi** [reply]
For the one that didn't interview? Yes, it did that successfully but never marked the interview as complete!
